### ISSUE TITLE
Chorus AIRBUS 3D test case 

### ISSUE THREAD

**Issue by prudhomm** [feelpp/toolbox#48]
add Chorus AIRBUS testcase to feelpp/toolbox

**Comment by prudhomm** [comment]
@jbwahl could you  update this one ?
perhaps it is best to move to v107 considering the complexity

**Comment by jbwahl** [comment]
indeed, there is some work to do with @vincentchabannes regarding the preconditioner strategy.
I guess it is more realistic to move it to v107
